index.html

```diff
@@ -491,13 +491,16 @@ <h2>Sharing theme</h2>
         x-cloak
         x-show="notice && notice.visible"
         @click="remove(notice && notice.id)"
+        @click.outside="remove(notice && notice.id)"
```

Comment:
This is problematic because it causes double clicks on hint to flash the hint and destroy it.

Also the `@click="resetHint"` on line 214 has the same problem and I think can just be completely removed at this point.

What do you think about something like:
- @click.outside - if within the first second of displaying it, ignore it.

Reply:
not as bad as I thought it was, as the 3rd click brings back the hint/

Reply:
I just removed it altogether - not really needed as it fades anyway